src/CPUPlayer.java

```diff
@@ -1,39 +1,36 @@
-import java.util.ArrayList;
+import java.util.*;
 
 class CPUPlayer {
     private static final int MAX_DEPTH = 2;
+    private static final int TIME_LIMIT = 5000;
```

Comment:
So it never goes further than depth 2?

Reply:
Dynamic depth is not ready

Reply:
yes but we can put max depth 3 and put the time limit lower so that it will explore part of the third depth

Reply:
No that is not how minimax works. As I told you this is just to make sure we don't go over the time limit. Minimax goes through the nodes leaves by leaves instead of levels by levels. So ATM if you put MAX_DEPTH = 3, it will go through about 40 / 160 possible moves.

Reply:
Oh ok yeah makes sense. if we do what Im saying, were gonna miss out on a bunch of the first level possible moves. Am I understanding correctly?